labkey/query.py

```diff
@@ -179,12 +180,16 @@ def delete_rows(
     :param query_name: table name to delete from
     :param rows: Set of rows to delete
     :param container_path: labkey container path if not already set in context
+    :param options: Additional parameters
```

Comment:
I feel like the overall python api docs need a rehaul at some point. For now though there are a couple of options:
1. Barebones comment as in PR now
2. Provide list of several common options in comment?
```
:param options: Additional parameters. Common examples include 
- auditUserComment, a string that can be used to provide a comment that will be attached to audit log records.
- auditBehavior, a string that can be used to override the default audit behavior for the table the query is acting on. The set of accepted values include: NONE, SUMMARY, and DETAILED.
```

Reply:
Yep, we're updating the Docs in the next couple of weeks. Joe is doing the bulk of the work and I'll be adding some additional stuff as well. I'll probably be assigning you to review my PRs.

Reply:
(Outdated thread)